### ISSUE TITLE
v1.11.0 causes webpack SplitChunksPlugin error

### ISSUE THREAD

**Issue by timiyay** [ef4/ember-auto-import#365]
Dependabot opened an update PR against several of our apps for `ember-auto-import@v1.11.0`, and our CI is failing with the following error:

```
/app/node_modules/webpack/lib/optimize/SplitChunksPlugin.js:803
								throw new Error(
								^

Error: SplitChunksPlugin: You are trying to set a filename for a chunk which is (also) loaded on demand. The runtime can only handle loading of chunks which match the chunkFilename schema. Using a custom filename would fail at runtime. (cache group: vendors)
    at /app/node_modules/webpack/lib/optimize/SplitChunksPlugin.js:803:15
    at SyncBailHook.eval (<anonymous>:7:16)
    at SyncBailHook.lazyCompileHook (/app/node_modules/tapable/lib/Hook.js:154:20)
    at Compilation.seal (/app/node_modules/webpack/lib/Compilation.js:1336:38)
    at /app/node_modules/webpack/lib/Compiler.js:675:18
    at /app/node_modules/webpack/lib/Compilation.js:1261:4
    at AsyncSeriesHook.eval [as callAsync] (<anonymous>:15:1)
    at AsyncSeriesHook.lazyCompileHook (/app/node_modules/tapable/lib/Hook.js:154:20)
    at Compilation.finish (/app/node_modules/webpack/lib/Compilation.js:1253:28)
    at /app/node_modules/webpack/lib/Compiler.js:672:17
    at _done (<anonymous>:9:1)
    at eval (<anonymous>:20:22)
    at /app/node_modules/webpack/lib/Compilation.js:1185:12
    at /app/node_modules/webpack/lib/Compilation.js:1097:9
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
```

This error is occurring when one of our addons uses dynamic imports load `Intl` polyfills for `ember-intl`, like so:

```js
await import('@formatjs/intl-getcanonicallocales/polyfill')
await import('@formatjs/intl-locale/polyfill')
await import('@formatjs/intl-pluralrules/polyfill')
await import('@formatjs/intl-numberformat/polyfill')
await import('@formatjs/intl-relativetimeformat/polyfill')
await import('@formatjs/intl-datetimeformat/polyfill')
```

**Comment by Turbo87** [comment]
same issue for us:

```
Error: SplitChunksPlugin: You are trying to set a filename for a chunk which is (also) loaded on demand. The runtime can only handle loading of chunks which match the chunkFilename schema. Using a custom filename would fail at runtime. (cache group: vendors)
    at /builds/front/qonto-register-js/node_modules/webpack/lib/optimize/SplitChunksPlugin.js:807:15
    at SyncBailHook.eval [as call] (eval at create (/builds/front/qonto-register-js/node_modules/tapable/lib/HookCodeFactory.js:19:10), <anonymous>:5:16)
    at SyncBailHook.lazyCompileHook (/builds/front/qonto-register-js/node_modules/tapable/lib/Hook.js:154:20)
    at Compilation.seal (/builds/front/qonto-register-js/node_modules/webpack/lib/Compilation.js:1336:38)
    at /builds/front/qonto-register-js/node_modules/webpack/lib/Compiler.js:675:18
    at /builds/front/qonto-register-js/node_modules/webpack/lib/Compilation.js:1261:4
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/builds/front/qonto-register-js/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
    at AsyncSeriesHook.lazyCompileHook (/builds/front/qonto-register-js/node_modules/tapable/lib/Hook.js:154:20)
    at Compilation.finish (/builds/front/qonto-register-js/node_modules/webpack/lib/Compilation.js:1253:28)
    at /builds/front/qonto-register-js/node_modules/webpack/lib/Compiler.js:672:17
    at _done (eval at create (/builds/front/qonto-register-js/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:7:1)
    at eval (eval at create (/builds/front/qonto-register-js/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:18:22)
    at /builds/front/qonto-register-js/node_modules/webpack/lib/Compilation.js:1185:12
    at /builds/front/qonto-register-js/node_modules/webpack/lib/Compilation.js:1097:9
    at processTicksAndRejections (internal/process/task_queues.js:79:11)
```

and we're also using those `@formatjs` packages, so it might be directly related.

**Comment by nightire** [comment]
we saw this error too, although we do not use `@formatjs` packages directly, we do use ember-intl as well.

**Comment by makepanic** [comment]
> we saw this error too, although we do not use `@formatjs` packages directly, we do use ember-intl as well.

Do you use dynamic imports?

**Comment by nightire** [comment]
> Do you use dynamic imports?

Yes, we did, but not for `@formatjs` packages.

**Comment by ef4** [comment]
Thanks for testing and reporting, I'm looking into this now.

**Comment by ef4** [comment]
This is going to turn out to be connected to https://github.com/ef4/ember-auto-import/pull/350, specifically https://github.com/ef4/ember-auto-import/blob/8c8bb1c6dc24b91a1be4dca33588cba19f193a15/packages/ember-auto-import/ts/webpack.ts#L130.  

That change was needed to make our rebuilds correct, because we don't want the names of those chunks to change from build-to-build.

**Comment by ef4** [comment]
This wasn't reproduced in our tests because you need to dynamically import two different modules that share a big-enough transitive module, causing webpack to make a common chunk for them.

**Comment by ef4** [comment]
Should be fixed in 1.11.1.

**Comment by timiyay** [comment]
Thanks for the fix!
